### ISSUE TITLE
Problem: PREMIS events importing fails when there's no eventOutcomeDetail element

### ISSUE THREAD

**Issue by replaceafill** [archivematica/Issues#967]
**Expected behaviour**

The PREMIS events importer works as expected.

**Current behaviour**

The `Load PREMIS events from metadata/premis.xml` job (added in https://github.com/archivematica/Issues/issues/710) fails when a `premis:event` element contains a `premis:eventOutcomeInformation` element with no `premis:eventOutcomeDetail` child.

According to the PREMIS spec [the `eventOutcomeDetail` element is optional](https://github.com/LibraryOfCongress/premis-v3-0/blob/3b79c30136c0f6ed673dd6ec9c9830d55730d889/xsd#L484).

**Steps to reproduce**

1. Run a transfer with a `metadata/premis.xml` file that includes a `virus check` `premis:event` element with a `premis:eventOutcomeInformation` like this (no `premis:eventOutcomeDetail` child):

```xml
    <premis:eventOutcomeInformation>
      <premis:eventOutcome>Pass</premis:eventOutcome>
    </premis:eventOutcomeInformation>
 ```

2. See the job failed and check its output to see an exception like this:

```
Instance of <class 'abc.PREMISEvent_outcome_informationElement'> has no attribute event_outcome_detail_note. Valid attributes are

event_outcome
event_outcome_informationTraceback (most recent call last):
  File "/src/MCPClient/lib/job.py", line 111, in JobContext
    yield
  File "/src/MCPClient/lib/clientScripts/load_premis_events_from_xml.py", line 854, in call
    job.set_status(main(job))
  File "/src/MCPClient/lib/clientScripts/load_premis_events_from_xml.py", line 810, in main
    events = get_elements(tree, "premis:event", event_element_factory, job.pyprint)
  File "/src/MCPClient/lib/clientScripts/load_premis_events_from_xml.py", line 309, in get_elements
    attributes = element_factory(element, printfn)
  File "/src/MCPClient/lib/clientScripts/load_premis_events_from_xml.py", line 432, in event_element_factory
    premis_element, "event_outcome_information", "event_outcome_detail_note"
  File "/src/MCPClient/lib/clientScripts/load_premis_events_from_xml.py", line 365, in get_premis_element_children_text_attribute
    if isinstance(getattr(e, attribute_name), six.string_types)
  File "/usr/local/lib/python2.7/dist-packages/metsrw/plugins/premisrw/premis.py", line 188, in __getattr__
    " are\n{}".format(self.__class__, attr_name, valid_attributes)
AttributeError: Instance of <class 'abc.PREMISEvent_outcome_informationElement'> has no attribute event_outcome_detail_note. Valid attributes are

event_outcome
event_outcome_information
```

**Your environment (version of Archivematica, operating system, other relevant details)**

qa/1.x

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by replaceafill** [comment]
I've created [this sampledata branch](https://github.com/artefactual/archivematica-sampledata/tree/dev/issue-967-fix-premis-import) to help testing this issue.

**Comment by evelynPM** [comment]
Fix verified on AM 1.11-qa (bionic)
